src/app/api/branching/branch/create/route.js

```diff
@@ -0,0 +1,63 @@
+import { NextResponse } from 'next/server';
+
+import { handleInsertBranchConnection, scheduleBranchDeletion } from '../../utils';
+
+// eslint-disable-next-line import/prefer-default-export
+export async function POST() {
+  try {
+    const start = performance.now();
+    const response = await fetch(
+      `https://console.neon.tech/api/v2/projects/${process.env.NEON_BRANCHING_DEMO_PROJECT_ID}/branches`,
+      {
+        method: 'POST',
+        headers: {
+          accept: 'application/json',
+          'content-type': 'application/json',
+          authorization: `Bearer ${process.env.NEON_BRANCHING_DEMO_API_KEY}`,
+        },
+        body: JSON.stringify({
+          endpoints: [
+            {
+              type: 'read_write',
+              autoscaling_limit_min_cu: 1,
+              autoscaling_limit_max_cu: 1,
```

Comment:
can we add `suspend_timeout_seconds: -1,` like in the original demo - https://github.com/neondatabase/branching-demo/blob/main/app/project/create/route.ts#L30C9-L30C37